pkg/cmd/cli/cmd/login/logout.go

```diff
@@ -123,25 +125,40 @@ func (o LogoutOptions) RunLogout() error {
 		return err
 	}
 
-	if err := client.OAuthAccessTokens().Delete(token); err != nil {
-		return err
+	errs := []error{}
+	serverErr := client.OAuthAccessTokens().Delete(token)
+	errs = append(errs, serverErr)
+
+	if configErr := deleteTokenFromConfig(*o.StartingKubeConfig, o.PathOptions, token); configErr != nil {
+		errs = append(errs, configErr)
+	} else if serverErr != nil {
+		// if token was not able to be deleted using api but was successfully removed
+		// from client config, log brief explanation that token was still removed locally
+		glog.V(1).Infof("%s\n\n", "An error ocurred deleting the token on the server. The token has been removed from your local configuration.")
+	}
+
+	err = kutilerrors.Flatten(kutilerrors.NewAggregate(errs))
+	if err != nil {
+		glog.V(1).Infof("%v\n", err)
 	}
 
-	newConfig := *o.StartingKubeConfig
+	fmt.Fprintf(o.Out, "Logged %q out on %q\n", userInfo.Name, o.Config.Host)
 
-	for key, value := range newConfig.AuthInfos {
-		if value.Token == token {
+	return nil
```

Comment:
The logic now is -- we fail to delete from API server, and we fail to delete from config, but we print the logout message and return `nil` so we exit successfully. Are both of these operations meant to be best-effort?